Common/Systems/Synchronization/Handlers/ElementalReverberationHandler.cs

```diff
@@ -0,0 +1,31 @@
+using PathOfTerraria.Common.Systems.ElementalDamage;
+using PathOfTerraria.Content.Buffs;
+using System.IO;
+
+namespace PathOfTerraria.Common.Systems.Synchronization.Handlers;
+
+/// <summary>
+/// Adds a stack of elemental echoing damage to an NPC. Should only be called from the client.
+/// </summary>
+internal class ElementalReverberationHandler : Handler
```

Comment:
Packets are content implementation details that would preferably go in the same file that uses them.